src/test/java/lotto/domain/lotto/LottoRankTest.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain.lotto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class LottoRankTest {
+
+    @DisplayName("당첨 번호 개수와 보너스 번호 포함 여부에 따라 로또 당첨 순위를 판별")
+    @ParameterizedTest
```

Comment:
`@ParameterizedTest`에도 예쁜 이름을 부여할 수 있다는 사실, 알고 계셨나요?
```suggestion
    @ParameterizedTest(name = "{0} 개의 숫자가 일치하고, 보너스 번호 일치 여부가 {1} 일 경우 {3} 당첨")
```